### PR TITLE
Adds Papa 10a cruise

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

**Comment by cdobs** [comment]
Cruise times can be reviewed in Papa 10a QLR (3203-01005)

File: cruise/CruiseInformation.csv [hunk 1]
```diff
@@ -331,3 +331,4 @@ TN422,Thomas G. Thompson,2023-08-13T00:00:00,2023-09-18T23:59:59,Cabled (rvdata)
 EK20230823,Elakha,2023-08-23T16:19:00,2023-08-23T20:09:00,Endurance cspp deployment CE01ISSP-00022
 AR76-03,Neil Armstrong,2023-08-28T10:00:00,2023-09-21T12:08:00,Irminger 10
 AT50-16,Atlantis,2023-09-21T15:40:00,2023-10-07T17:00:00,Endurance 19 moorings profilers and gliders
+SU20231023,Pacific Surveyor,2023-10-23T14:00:00,2023-10-24T12:00:00,Papa 10a glider deployments
```
Comment (rtravis-whoi) [inline cruise/CruiseInformation.csv:334]: looks good

**Review by rtravis-whoi — APPROVED** [review]

File: cruise/CruiseInformation.csv (hunk 1 above)
Comment (cooleyky) [inline cruise/CruiseInformation.csv:334]: looks good

**Review by cooleyky — APPROVED** [review]
Posted comments instead of approval earlier. Looks good.

**Pull request merged**